frontend/services/index.ts

```diff
@@ -12,7 +12,8 @@ const sendRequest = async (
   path: string,
   data?: unknown,
   responseType: AxiosResponseType = "json",
-  timeout?: number
+  timeout?: number,
+  includeFullAxiosError?: boolean
```

Comment:
Nice. Any particular reason for this or just to add flexibility and observability to the error handling pattern?